src/main/java/eiteam/esteemedinnovation/commons/util/ReflectionHelper.java

```diff
@@ -85,7 +85,7 @@ private static void setAccessible(Method method) {
     }
 
     public static void init() {
-        FMLLog.info("[EI] Getting some fields through reflection.");
+        EsteemedInnovation.logger.info("[EI] Getting some fields through reflection.");
```

Comment:
`[EI]` should be removed from this, because it says `[esteemedinnovation]` in each log entry